src/Scope.zig

```diff
@@ -13,6 +13,13 @@ pub const AliasList = std.ArrayListUnmanaged(struct {
     name: []const u8,
 });
 
+// Associates a container (structure or union) with its relevant member functions.
+pub const ContainerMemberFns = struct {
+    container_decl: ast.Node,
+    member_fns: std.ArrayListUnmanaged(*ast.Payload.Func),
+};
+pub const ContainerMemberFnsHashMap = std.AutoHashMapUnmanaged(aro.QualType, ContainerMemberFns);
```

Comment:
Might as well make this a `AutoArrayHashMapUnmanaged` since you're going to be iterating on it.